client/lib/utils/coerce-values.js

```diff
@@ -1,3 +1,5 @@
+let coerceFormValues;
```

Comment:
Sadly, it is, because of [this ESLint rule](http://eslint.org/docs/rules/no-use-before-define).

`coerceFormValues` is called from `coerceValue`, and `coerceValue` is called from `coerceFormValues`. Either we disable the ESLint rule, or `coerceFormValues` needs to be declared on the top of the file.